e2e/tests/heavy_writing_and_recurring_jobs.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation    Negative Test Cases
+Resource    ../keywords/volume.resource
+Resource    ../keywords/node.resource
+Resource    ../keywords/common.resource
+Resource    ../keywords/recurring_job.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+
+*** Test Cases ***
+Volume Attached Node Reboot While Heavy Writing And Recurring Jobs Exist
```

Comment:
What's the purpose to test this with recurring jobs?

I feel `Reboot volume node while heavy write IO` good enough?

Reply:
I see. Let's keep the same test plan first.

Reply:
cc @khushboo-rancher It would be great if related context or tickets can be referenced in the confluence page.